tips-tricks/TroubleshootingandTips.rst

```diff
@@ -679,20 +679,22 @@ You need to load data from a structured datastore.
+a. **GetTableData** leverages JDBC to pulls data from the source into the flowfile within NiFi. This content will then need to be pushed to HDFS (via PutHDFS processor) to the landing area in HDFS.
 
 b. **ImportSqoop** executes a import-sqoop call which pull the content from the source directly to HDFS. 
 
-In general, it is recommended to use the ImportSqoop processor due to performance and minimal IO. Using GetTableData uses the edge node (where Nifi is running) as a middle-man. Also, ImportSqoop runs a MapReduce job which allows you to tune the performance to the data being ingested. For example, if you are loading a reference table, a single mapper will be sufficient while billions of rows would require multiple mappers.
+In general, it is recommended to use the ImportSqoop processor due to performance and minimal IO. Using GetTableData uses the edge node (where Nifi is running) as a middle-man. Also, ImportSqoop runs a MapReduce job which allows you to tune the performance to the data being ingested. For example, if you are loading a reference table, a single mapper will be sufficient, while billions of rows would benefit from multiple mappers.
 
-The GetTableData should be used when the data being pulled is small and will be used for further processing. For example, getting the list of tables within a database so additional processing can be done.
+The GetTableData should be used when the data being pulled is small. Other use cases are when certain pre-processing steps are required that benefit from being on the edge node. For instance, if the edge node resides behind a firewall and PII fields need to be masked before pushing to a more open HDFS environment.
 
-One important consideration when using ImportSqoop is that the standard-ingest template needs to be updated. The default instance of the template assumes the content is within the flowfile. The major changes are:
+One important consideration when using ImportSqoop is that the out-of-box standard-ingest NiFi template needs to be updated. The default instance of the template assumes the content to be ingested is within the flowfile. To allow standard-ingest to accept data from a ImportSqoop processor, the following changes need to be made to the standard-ingest template:
 
 1. Remove the PutHDFS processor from the flow
 
 2. Update the "Create Feed Partition" processor to point to the target location of ImportSqoop
 
-3. Create a new archive processor which will archive data from HDFS
+3. Create a new archive processor which will archive data from HDFS. One option is use the Hadoop streaming tool to take the files residing in the target location of ImportSqoop and compress the data to the archive directory.
+
+It is important to note that any other templates (ex. Data Ingest) that output to standard-ingest would need to be updated because the changes above assumes data resides in HDFS. In general, adding a PutHDFS processor will be sufficient.
```

Comment:
Please modify 'ex.' to 'e.g.' to make it clear that we are referring to an example template.